NET-specifics/data-binding.md

```diff
@@ -1,2 +1,213 @@
 # Data Binding
 
+# Data Binding Overview
+
+The graphical user interface of OSPSuite uses [Winforms](https://en.wikipedia.org/wiki/Windows_Forms) as its UI framework and the [Devexpress Winforms Component](https://docs.devexpress.com/WindowsForms/7874/winforms-controls) to provide additional controls and functionalities. In order to display the data on the controls, a data binding framework has been developed. The data binding is part of two separate repositories, [OSPSuite.DataBinding](https://github.com/Open-Systems-Pharmacology/OSPSuite.DataBinding) and the DevExpress specific [OSPSuite.DataBinding.DevExpress](https://github.com/Open-Systems-Pharmacology/OSPSuite.DataBinding.DevExpress). In almost all cases, simple bindings can be created to implement two way updating of view and objects.
+
+# Data Binding Example
+
+## DTO (Data Transfer Object)
+
+Although we could bind to the properties of an object directly, we generally prefer to use DTOs [Data Transfer Object](https://en.wikipedia.org/wiki/Data_transfer_object) as an intermediate step, in order to be able to structure our data in a manner that makes more sense in displaying. Let's take as an example the binding to a simple TextEdit. We are going to base our example initially to binding a simple float value to the text edit:
+
+So let's say we have defined in our view a TextEdit.
+
+```
+private DevExpress.XtraEditors.TextEdit myValueTextEdit;
+```
+
+In this example we want to bind the TextEdit to a `float` value. We will create a very simple DTO to hold that value - later on we will also add a validation check for this value. But let's start with the simplest possible form:
+
+```
+public class MyValueDTO
+{
+      public float MyValue { get; set; }
+}
+```
+
+## Screenbinder
```

Comment:
In this section, since you are describing the initialization, we should also mention that for every screenbinder that is initialized, you need to call _screenBinder.Dispose too.

I always create a method like disposeBinders and then I go into the designer file and call it from the Forms method 'Dispose'.

Reply:
description for disposing added